### PR TITLE
Us3 feedback v2

### PR DESCRIPTION
**Pull request by teatimes**
Made database tests, changed some components and added new functions.

### CODE REVIEW

File: src/client/components/Message.js [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
+          disabled
+          icon={<ActionSchedule style={{width: '18px', height: '18px', margin: '0px'}} />}
         />
-    	</Paper>
+      </Paper>
     )
   }
 }
 
-Message.propTypes = {
```
Comment (weedle1912) [inline src/client/components/Message.js]: This line is needed. "Message.propTypes", not just "propTypes", for anything to render in browser.

**Review by weedle1912 — CHANGES_REQUESTED** [review]
Errors when using UI to send message:
- Unhandled rejection SequelizeDatabaseError: column Feedback.lectureId does not exist
- Unhandled rejection SequelizeDatabaseError: column "messageId" does not exist
- Unhandled rejection TypeError: result.setLecture is not a function

**Review by teatimes — COMMENTED** [review]
Fixed requested changes

**Review by weedle1912 — CHANGES_REQUESTED** [review]
1) When connected to database:
- postgres_1| ERROR:  column "userId" does not exist at character 80
- viestinta_node| Unhandled rejection SequelizeDatabaseError: column "userId" does not exist

2) When sending messages in UI:
- postgres_1| ERROR:  column "lectureId" of relation "Messages" does not exist at character 23
- viestinta_node| Unhandled rejection SequelizeDatabaseError: column "lectureId" of relation "Messages" does not exist

Everything else is OK. These errors are only visible in terminal.

**Comment by odgaard** [comment]
Approved, even though it decreases the code coverage from 98.66% to 93.85%.  Tested on my Ubuntu laptop and Drone.

**Pull request merged**